### ISSUE TITLE
.tmux/plugins is deleted on tmux start - Ubuntu 23.10

### ISSUE THREAD

**Issue by jamesfreeman959** [gpakosz/.tmux#709]
Hi there

I love your work first of all - I have it working perfectly on macOS! I've just tried to port my setup to my main Linux server (running Ubuntu Server 23.10 at this time) and I'm experiencing some odd problems.

I install your .tmux framework in the instructed manner (have checked all versions of requirements), and then cloned tpm into .tmux/plugins as instructed.

I then run tmux for the first time. The status bar reports that plugins are being updated or installed. When this completes, the .tmux/plugins directory no longer exists - even tpm has been deleted!

I'm struggling to debug this but if you can advise on the steps I need to run to assist, I'll grab whatever output is needed.

Many thanks

James

**Comment by jamesfreeman959** [comment]
Ok - I now can't reproduce the problems I was seeing, so closing this. A couple of things I noted which helped me in (attempting) debugging this as far as I did:

1. If you install tpm and no plugins are enabled, then default behaviour is to uninstall tpm - hence the whole plugins directory disappears when you run tmux. I had removed all plugins for debugging purposes in case one was causing the issue.
2. Run `tmux -v` or `tmux -vv` to get debug logging enabled - it's logged to the current directory when you ran tmux from. Look in the `tmux-server-<pid>.log` file for in depth details of what is going on.
3. I presume I had a problem with the install of one plugin as I was seeing the plugins being momentarily installed, then all deleted. I now can't reproduce this so I can't give a root cause for this - however I hope the above helps someone!
